flutter_app/lib/screens/home_screen.dart

```diff
@@ -21,6 +24,9 @@ class HomePage extends StatelessWidget {
           DroneInformation(
             getDroneInformation: GetDroneInformation(comm: comm),
           ),
+          LogsList(
```

Comment:
Remove once you're done testing, there will be a task to add the widgets we've developed so far to main in the future